contracts/service-registry/src/state.rs

```diff
@@ -21,7 +21,7 @@ use crate::ContractError;
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 pub struct Service {
     pub name: String,
-    pub service_contract: Addr,
+    pub coordinator_contract: Addr,
```

Comment:
did the purpose of this field change? What was it and why is it not needed anymore?

Reply:
I was told that it was always meant for the coordinator. Since the name was not finalized when creating the instantiation of service registry, it was named as `service_contract`.